### PR TITLE
www-apps/gitea: add 1.21.4

### PR DESCRIPTION
**Pull request by bekcpear**
**tested on amd64**

Update:

Don't show the annoying warnings about the breaking from version 1.20 to 1.21 if the currently installed version is already greater than or equal to 1.21.0, or if no gitea is currently installed.

### CODE REVIEW

File: www-apps/gitea/gitea-1.21.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+		eerror "to higher priority path '/etc/tmpfiles.d/gitea.conf', and correct it with"
+		eerror "the right User and Group value (see tmpfiles.d(5) for details), then execute:"
+		eerror "  # systemd-tmpfiles --create /etc/tmpfiles.d/gitea.conf"
+		eerror "to install it."
+	fi
+
+	if [[ -n ${REPLACING_VERSIONS} ]] ; then
+		local -i major minor
+		IFS="." read -r major minor _ <<<"$REPLACING_VERSIONS"
+		if (( major == 1 && minor < 21 )); then
```
Comment (dlan17) [inline www-apps/gitea/gitea-1.21.4.ebuild:142]: this is kind of tedious.. can you take a look at 'version_is_at_least' (or version_compare, in versionator.eclass)? it should work for you

Comment (arkamar) [reply www-apps/gitea/gitea-1.21.4.ebuild:142]: `versionator.eclass` does not support EAPI 8, use `ver_test`, see https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#version-comparison-ver-test for more information.

**Comment by hydrapolic** [comment]
Tested on amd64, works fine, thanks!

File: www-apps/gitea/gitea-1.21.4.ebuild (hunk 1 above)
Comment (bekcpear) [reply www-apps/gitea/gitea-1.21.4.ebuild]: Corrected, thank you very much for pointing this out!

**Pull request closed without merging**